manual.md

```diff
@@ -16,27 +14,43 @@ The Radworks Treasury is controlled by a tokenized governance mechanism ([Compou
 Currently, these actions include:
 
 - Treasury distributions of assets (e.g. RAD, ETH, and USDC)
-- Upgrades to the on-chain governance system (Compound Governor)
-- Changes to parameters of Treasury-governed contracts (e.g. Drips)
+- Upgrades to the on-chain governance system (e.g. Governance, Timelock)
+- Changes to parameters of Treasury-governed protocols (e.g. Drips)
 
 To learn more about token governance and how Radworks’ on-chain governance system works, [see here](https://wiki.tally.xyz/docs/compound-governor).
 
 ### Off-chain
 
-In addition to the on-chain governance system, Radworks manages a set of off-chain processes and resources that define the rules for participating in the DAO. These processes and resources are maintained off-chain, but updates to these documents still require formal community approval via Snapshot vote. An off-chain review process is also required for all proposals before they can be submitted on-chain.
+In addition to the on-chain governance system, Radworks manages a set of off-chain processes and resources that define the rules for participating in governance. These processes and resources are maintained off-chain, but updates to these documents still require formal community approval via Snapshot vote. An off-chain review process (see [Formal Review](#formal-review) is also required for all proposals before they can be submitted on-chain.
```

Comment:
Maybe change to: 
In addition to the on-chain governance system, Radworks utilizes a formal off-chain review process to gain community consensus around making changes to off-chain process and resources, as well as to review proposals intended to be submitted on-chain. See Formal Review (link) below. 

updates to these documents still require community approval via Formal Review (link), including a Snapshot poll. All proposals tha

Reply:
Did you mean to complete this sentence?

Reply:
Shoot sorry not sure what happened there. I don't think that last incomplete sentence was supposed to be there as I tried to summarize the actions in the sentence above.